### ISSUE TITLE
Nord look fails to install

### ISSUE THREAD

**Issue by danielc103** [regolith-linux/regolith-desktop#259]
```shell
sudo apt install regolith-look-nord
Reading package lists... Done
Building dependency tree
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 regolith-look-nord : Depends: nordic but it is not installable
E: Unable to correct problems, you have held broken packages.
```

Installed Nordic locally and globally from repo and set theme in tweaks. Ran again, still get same error.

`sudo apt install nordic` returns package not available and has no installation candidate.

Installed   `regolith-look-ubuntu` with no issues.

**Comment by Fenton-Fenton** [comment]
I've also had the same issue on a fresh install. Problem seems to apply to all themes apart from the default and ubuntu.

**Comment by kgilmer** [comment]
This is fixed in 1.3.1
